templates/wordpress/1/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+wordpress:
+  image: wordpress
+  links:
+    - db:mysql
```

Comment:
# Set the link to database service
-- - db:mysql
++ - ${wordpress_db_service}:mysql